inc/admin.php

```diff
@@ -509,3 +515,35 @@ function clean_orphaned_posts( $option, $post_type ) {
 
 	return true;
 }
+
+/**
+ * Check if ads.txt file already exists in the server
+ *
+ * @return void
+ */
+function adstxts_check_for_existing_file() {
+	$home_url_parsed = wp_parse_url( home_url() );
```

Comment:
Security: This will need both a nonce and a capability check, similar to that used in the save function:

https://github.com/10up/ads-txt/blob/dc18245c6add3f0eaeaafca83c2dc1d88ecd209c/inc/save.php#L19-L20